### PR TITLE
OSDOCS-6143: Added vCenter requirements section to vSphere UPI docs

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-6143](https://issues.redhat.com/browse/OSDOCS-6143)

Version(s):
4.14

Previews:
* [Installing a cluster on vSphere with user-provisioned infrastructure](https://60436--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-installer-infra-requirements_installing-vsphere)
* [Installing a cluster on vSphere with network customizations](https://60436--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-vsphere-installer-infra-requirements_installing-vsphere-network-customizations)
* [Installing a cluster on vSphere in a restricted network with user-provisioned infrastructure](https://60436--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-restricted-networks-vsphere.html#installation-vsphere-installer-infra-requirements_installing-restricted-networks-vsphere)
* [Installing a cluster on VMC with user-provisioned infrastructure](https://60436--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-vsphere-installer-infra-requirements_installing-vmc-user-infra)
* [Installing a cluster on VMC with user-provisioned infrastructure and network customizations](https://60436--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-network-customizations-user-infra.html#installation-vsphere-installer-infra-requirements_installing-vmc-network-customizations-user-infra)
* [Installing a cluster on VMC in a restricted network with user-provisioned infrastructure](https://60436--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-restricted-networks-vmc-user-infra.html#installation-vsphere-installer-infra-requirements_installing-restricted-networks-vmc-user-infra)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by sgaoshang** [comment]
We should also remove 
```
"Virtual machine"."Edit Inventory"."Create new"
"Virtual machine"."Edit Inventory"."Remove"
```
in “Roles and privileges required for installation in vCenter graphical user interface (GUI)“ to align with "Roles and privileges required for installation in vSphere API"

**Comment by dfitzmau** [comment]
> We should also remove
> 
> ```
> "Virtual machine"."Edit Inventory"."Create new"
> "Virtual machine"."Edit Inventory"."Remove"
> ```
> 
> in “Roles and privileges required for installation in vCenter graphical user interface (GUI)“ to align with "Roles and privileges required for installation in vSphere API"

Thanks, @sgaoshang . I updated the tables to remove these privileges for UPI docs sets. Please let me know your thoughts.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 1]
```diff
@@ -90,7 +133,7 @@ An additional role is required if the installation program is to create a vSpher
 |`Network.Assign`
 
 |Virtual Machine Folder
-|Always
+|Always.
```
Comment (stevsmit) [inline modules/installation-vsphere-installer-infra-requirements.adoc:136]: None of the other "Always" in the same category ("When required") have a period at the end.

Comment (dfitzmau) [reply modules/installation-vsphere-installer-infra-requirements.adoc]: Thanks. Fixed.

Comment (dfitzmau) [reply modules/installation-vsphere-installer-infra-requirements.adoc]: Thanks. Fixed.

**Pull request merged**